Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
> and then sync stuff back to the main k8s.io/metrics repo as per the staging sync process

this part is not easy. I'll need to reconfigure the publishing robot and i don't know if it will work as expected :)